frontend/src/metabase/css/admin.module.css

```diff
@@ -7,60 +7,60 @@
 }
 
 /* utility to get a simple common hover state for admin items */
-.HoverItem:hover {
+:global(.HoverItem:hover) {
```

Comment:
Yeah just to reiterate, it'd be great if we could stick to one style either using parentheses or not if that's possible